README.md

```diff
@@ -203,7 +116,70 @@ html_context = {
 
 By enabling analytics we will be able to collect information on number of visits to each documentation page. It will also trigger the addition of a `Was this page helpful?` component at the bottom of each documentation page, so users will be able to provide yes/no feedback for each page.
 
-<img width="538" alt="Screenshot 2022-11-14 at 11 28 11 AM" src="https://user-images.githubusercontent.com/23662430/201715694-1e75b1fb-875b-4137-b9f3-93d1e1894f5a.png">
+![](tests/js/snapshots.test.js-snapshots/footer-includes-page-analytics-1-linux.png)
+
+If you do not enable analytics, no data will be collected and the `Was this page helpful?` component will not show.
+
+## Migrate from old Pytorch theme to new theme
+
+In qiskit-sphinx-theme 1.13, we migrated to a new Sphinx theme based on Furo, which is used by pip, Black, and attrs documentation. See https://github.com/Qiskit/qiskit_sphinx_theme/issues/232 for the motivation.
+
+qiskit-sphinx-theme 1.13 continues to support the legacy Pytorch theme, but support will be removed in version 2.0.
+
+To migrate:
+
+1. In `conf.py`, ensure that `"qiskit_sphinx_theme"` is in the `extensions` list.
+2. In `conf.py`, set `html_theme = "qiskit"` rather than `"qiskit_sphinx_theme"`.
+3. In `conf.py`, remove all `html_theme_options`.
+4. In `conf.py`, remove `expandable_sidebar` from `html_context`, if set. If it was set, follow the below section [How to migrate expandable_sidebar](#how-to-migrate-expandablesidebar).
+5. Render the docs and check that everything looks how expected. If not, please open a GitHub issue or reach out on Slack for help.
+
+Reminder: you do not have to migrate to the `qiskit` theme until qiskit-sphinx-theme 2.0. For example, you can migrate but rollback to `qiskit_sphinx_theme` if you discover there is an issue.
```

Comment:
I don't particularly like this sentence, I think we should be encouraging people to migrate as soon as possible before 2.0 and then raise concerns/bugs with us sooner rather than later. I worry that the tone of this sentence will lead people to not switch over until they absolutely have to for 2.0 and we'll have a bunch of issues to deal with at that time which could've been flagged earlier